client/components/editable.js

```diff
@@ -15,6 +15,7 @@ function Editable({ edited,
   onRevert = () => {} }) {
 
   const [ state, setState ] = useState({content, reminders});
+  content = state.content;
```

Comment:
This might need to be a bit more involved - in the cancel function we're checking
```if (state.content !== content) {```
which if we do this, will never return true.

I think the intention was to prevent you navigating away from the page before the content has saved. The save is triggered by calling the parent `onChange` in the use effect, so we might want to wait for that to return before updating the content?

something like
```Promise.resolve()
  .then(() => onChange(state))
  .then(() => content = state.content);
```